product/views.py

```diff
@@ -0,0 +1,134 @@
+from django.views import View
+from django.http  import JsonResponse
+
+from .models import *
+
+class ProductListView(View):
```

Comment:
productdetailview를 참고해서 수정해주세요~
productslist 보다는 products 만으로 충분할 것 같습니다.
```suggestion
class ProductsView(View):
```